docs/developing/ios.md

```diff
@@ -12,173 +12,28 @@ skipIntros: true
   />
 </head>
 
-This guide covers how to run and debug Ionic apps on iOS simulators and devices using [Capacitor](../reference/glossary.md#capacitor) or [Cordova](../reference/glossary.md#cordova). iOS apps can only be developed on macOS with Xcode installed.
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
 
-There are two workflows for running Ionic apps on iOS:
+:::info Looking for the legacy iOS guide?
 
-- [Running with Xcode](#running-with-xcode)
-- [Running with the Ionic CLI](#running-with-the-ionic-cli)
+The Developing for iOS guide has officially migrated to the Capacitor docs: https://capacitorjs.com/docs/ios. Looking for the legacy docs? [Click here](/docs/v6/developing/ios).
```

Comment:
- This link, `https://capacitorjs.com/docs/android`, should be inline to be accessible compliant.
- "Click here" also is too vague to be accessible compliant.

This [article](https://business.scope.org.uk/article/how-to-write-better-link-text-for-accessibility) is a good read to understand why links should be set in a certain way for web accessibility.